ui/app/components/secure-variable-form.js

```diff
@@ -77,9 +77,19 @@ export default class SecureVariableFormComponent extends Component {
   async save(e) {
     e.preventDefault();
     try {
+      const nonEmptyItems = this.keyValues.filter(
+        (item) => item.key && item.value
```

Comment:
`nit`: This doesn't factor for whitespace. I set the key to be ` ` and the value to be ` `. And this is still valid. Maybe add `trim`?

Reply:
Hm, I think this is probably safe to do for the key, but I'm going to avoid it for value.

I think there's a world where someone wants a value of a whitespace character that would otherwise be trimmed for their secure variable, and we shouldn't necessarily nuke that. 

cc @angrycub  / @tgross in case there's a good reason to not trim keys as well.